src/codegen/templates/library_interpreter/double_c_function_call.py.mako

```diff
@@ -19,7 +20,9 @@
                     cfunc.argtypes = [
                         ${', '.join(get_argument_types(function)) | wrap(24, 24)}]
 
+%if function.function_name not in INCLUDE_STRING_BUFFER_SIZE_FUNCTIONS:
         temp_size = 0
+%endif
```

Comment:
For the size hint functions, set `temp_size = size_hint` rather than modifying the `size_hint` parameter.